.github/workflows/release.yml

```diff
@@ -0,0 +1,263 @@
+name: Release
+
+on:
+  # push:
+  #   branches: [auto-release-workflows]
+  workflow_dispatch:
+
+jobs:
+    
+  release:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      R_REMOTES_NO_ERRORS_FROM_WARNINGS: true
+      RSPM: "https://packagemanager.rstudio.com/cran/__linux__/focal/latest"
+
+    steps:
+      - name: Setup Git repository
+        uses: actions/checkout@v4
+
+      - name: Update Ubuntu packages
+        run: sudo apt-get update
+        
+      - name: Setup R
+        uses: r-lib/actions/setup-r@v2
+        with:
+          r-version: 'release'
+
+      - name: Setup pandoc
+        uses: r-lib/actions/setup-pandoc@v2
+        with:
+            pandoc-version: '2.14.0.3'
```

Comment:
This is an outdated version of pandoc. The current version is '2.17.1' but instead of supplying the specific version, which will always need to be updated, we could delete lines 31--32. I tested it out in my sandbox and removing those two lines still allows you to use pandoc.